docs/reference/search/suggesters/context-suggest.asciidoc

```diff
@@ -239,24 +271,22 @@ PUT services/service/1
 [float]
 ===== Geo location Query
 
-Like in the configuration, querying with a geo location in context, the geo location
-query supports all representations of a `geo_point` to define the location. In this
-simple case all precision values defined in the mapping will be applied to the given
-location.
+Suggestions can be filtered and boosted with respect to how close they are to one or
+more geo points. The following filters suggestions that fall within geo point:
```

Comment:
OK so you're using geohashes here - worth explaining how a geopoint gets translated internally.